app/src/main/java/com/example/pocketcloset/Fragments/WardrobeFragment.java

```diff
@@ -1,65 +1,224 @@
-     * @param param1 Parameter 1.
-     * @param param2 Parameter 2.
-     * @return A new instance of fragment CameraFragment.
-     */
-    // TODO: Rename and change types and number of parameters
-    public static WardrobeFragment newInstance(String param1, String param2) {
-        WardrobeFragment fragment = new WardrobeFragment();
-        Bundle args = new Bundle();
-        args.putString(ARG_PARAM1, param1);
-        args.putString(ARG_PARAM2, param2);
-        fragment.setArguments(args);
-        return fragment;
-    }
 
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (getArguments() != null) {
-            mParam1 = getArguments().getString(ARG_PARAM1);
-            mParam2 = getArguments().getString(ARG_PARAM2);
-        }
-    }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_camera, container, false);
+        return inflater.inflate(R.layout.fragment_wardrobe, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        rvWardrobe = view.findViewById(R.id.rvClothes);
+        toggleButton = view.findViewById(R.id.toggleButton);
+
+        allClothing = new ArrayList<>();
+        adapter = new ClothingAdapter(getContext(), allClothing);
+
+        rvWardrobe.setAdapter(adapter);
+        // set the layout manager on the recycler view
+        rvWardrobe.setLayoutManager(new LinearLayoutManager(getContext()));
+
+        int numberOfColumns =2;
+        GridLayoutManager gridLayoutManager = new GridLayoutManager(getContext(), numberOfColumns);
+
+        rvWardrobe.setLayoutManager(gridLayoutManager);
+
+        swipeContainer = (SwipeRefreshLayout) view.findViewById(R.id.swipeContainer);
+         //Setup refresh listener which triggers new data loading
+       swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                adapter.clear();
+                queryClothes();
+                adapter.notifyDataSetChanged();
+                swipeContainer.setRefreshing(false);
+            }
+        });
+       swipeContainer.setColorSchemeResources(android.R.color.holo_blue_bright,
+               android.R.color.holo_green_light,
+                android.R.color.holo_orange_light,
+                android.R.color.holo_red_light);
+
+
+
+       toggleButton.addOnButtonCheckedListener(new MaterialButtonToggleGroup.OnButtonCheckedListener() {
+           @Override
+           public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, boolean isChecked) {
+               if (checkedId == R.id.btnTop) {
```

Comment:
It looks like all these if statements are doing the same thing, and there's a cascade of them. My I recommend
```
String clothes = null;
switch(checkedId) {
  case R.id.btnTop:
    clothes = "Top";
    break;
    //etc
}

if (clothes != null) {
  adapter.clear();
  queryClothes(clothes);
  adapter.notifyDataSetChanged();
}
```

You can even take this further, since you know all of the ids and strings ahead of time
```
private static final Map<Integer, String> sButtonIdToClothingQuery = new HashMap();

static {
  sButtonIdToClothingQuery.put(R.id.btnTop, "Top");
  // etc
}

...

String clothes = sButtonIdToClothingQuery.get(checkedId);
if (clothes != null) {
  adapter.clear();
  queryClothes(clothes);
  adapter.notifyDataSetChanged();
}
```